packages/webdriverio/src/middlewares.js

```diff
@@ -36,12 +36,17 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
 
                     return await fn(commandName, commandFn).apply(this, args)
                 }
-                if (typeof error === CustomRequestError) {
-                    throw new Error(`
-                    Can't call ${commandName} on element with selector "${this.selector}" because:
-                    ${error}
-                    `)
+
+                if (error instanceof CustomRequestError) {
+                    if (error.message.includes(`"${this.selector}"`)) {
+                        const regex = /call (.*?) command/g
+                        error.message = error.message.replace(regex, `call ${commandName} command`)
```

Comment:
What is being replaced here?

Reply:
Actually, webdriverio sends 3 commands on `setValue` or 2 commands on `click` action. So, before `error` will be thrown, the number of iteration will be equal to number of commands.

For example for `setValue`:
first iteration: `elementClear` command
second: `clearValue ` command
third: `setValue` command

I need to show only last one command (`setValue` in particular case).
So, if `error.message` has already had `selector` but error still was not thrown then it is next iteration, and we need replace previous command with current and so on. 

message on first iteration: `Can't call elementClear commands on element with ....`
message on second iteration: `Can't call clearValue commands on element with ....`
message on last iteration: `Can't call setValue commands on element with ....`